src/prefect/client/client.py

```diff
@@ -895,25 +899,46 @@ def set_task_run_state(
 
         Raises:
             - ClientError: if the GraphQL mutation is bad for any reason
+
+        Returns:
+            - State: the state the current task run should be considered in
         """
         mutation = {
             "mutation($state: JSON!)": {
                 with_args(
-                    "setTaskRunState",
+                    "setTaskRunStates",
```

Comment:
Similar comment as above for constructing this query in a possibly more readable way.